website/content/docs/api-clients/api/index.mdx

```diff
@@ -82,6 +82,15 @@ The following method conventions are used within Boundary's API:
 
 `DELETE` is used for deleting a specific resource, and is only used against a particular resource path.
 
+## HTTP headers
+
+Clients that make requests to the controller API can inspect HTTP response headers to understand the configured limits and current usage.
+Responses may contain the following headers:
+
+- `RateLimit` - Provides the current limit, number of remaining requests, and the time at which the quota will reset for the limit that is closest to being exhausted for the requested resource and action.
+- `RateLimit-Policy` - Describes the limits for the requested resource and action.
+- `x-correlation-id` - Indicates that an API request that came in to Boundary is also related to an audit event in Vault.
```

Comment:
Might want to clarify what this is a bit, I am not sure on the best wording but:

If the user provides a `x-correlation-id` header in the HTTP request, Boundary will log that value in all audit events related to the request. It will also use that value as the `x-correlation-id` header when making requests to Vault, which will allow users to correlated Boundary and Vault Audit logs.

One thing to note here, this will eventually be for ALL external systems not just Vault. It just so happens that our first iteration of this only really targets Vault.